app/src/main/AndroidManifest.xml

```diff
@@ -40,32 +41,25 @@
             android:launchMode="singleTask"
             android:screenOrientation="portrait"
             android:theme="@style/Theme.Crf.Main">
+        </activity>
 
-            <!-- Intent filter to receive the Fitbit auth token -->
-            <!-- Remove the houston fitbit schema after sage has completed their integration -->
+        <activity android:name="net.openid.appauth.RedirectUriReceiverActivity"
```

Comment:
This seems a bit unnecessary to use a 3rd party library for just an intent filter, no?

Reply:
This is basically the blessed Google OAuth2 library and it does do some nice things like handling the backstack state nicely and merging updates to Authorization and Access state and persisting it. We could roll our own but I prefer using a library that supports the full feature set of OAuth2 correctly.

We currently aren't using these features, but if we want to have the app access the API (instead of just Bridge server) it supports things like PKCE and helpers like performActionWithFreshTokens

Reply:
Oh yea, it also handles Browser selection, preferring CustomTabs over the fallback options which I can't remember.

Reply:
Gotcha, that is a nice to have.  Please put that comment in the xml, and then you are good to merge.